doc/deploy/android.md

```diff
@@ -1,33 +1,129 @@
+  from the command line.  Wait for it to download the components.
+  1. Keep waiting
+  1. Check the boxes next to Android 8.0(Oreo) and any others leading up to the latest--you'll see that that's API 
+  level 26, which we target at the moment. 
+  1. Click on the "tools" tab. Make sure "Android SDK Platform Tools", "Android SDK Tools", and
+  "Android SDK Build Tools" are checked. If you don't have access to an Android phone, make sure
+  the Emulator is also installed. Click "apply"
 
-3. (PVTrAck 2+) `ionic build android --prod --release`
+### Java 1.8
+  1. `$brew update`
+  1. `$brew tap caskroom/versions`
+  1. `brew search java`
+  1. `brew cask install java8`
+  1. If you can figure out why this app requires an old version of Java, please
+  fix it.
 
-    Troubleshooting: if you get an error related to whether this is an Android project, try the following:
+### Gradle
+  1. `$brew install gradle`
+  1. Follow any additional commands to set Gradle up.
 
+### Node
+  1. Install node: `$brew install node`.
+  1. Then use it actually install our packages:`$npm install` (You probably
+  already did this when doing work on the application, but it can't hurt to
+  update if you pulled from master)
+
+## Ready to build?
+  1. Open `config.xml`: On line 2, increment `android-versionCode`
+  with your appropriately chosen new version number.
+  1. You may make a debug build first to try the changes to your application on
+  an emulator or phone: `$ionic cordova build android`.
+  1. If you have an Android: Plug it into the computer with a USB cable and enable USB
+  debugging. Side-load like so, replacing the paths with your own:
+ `~/Library/Android/sdk/platform-tools/adb install /Users/my_name/pvta-multiplatform/platforms/android/build/outputs/apk/debug/android-debug.apk
+  1. Obtain the PVTrAck signing key. See [the keys doc](android-keys.md) for more info.
+  Place it in the `platforms/android` directory. The `jks` extension is gitignored by default.
+  1. Obtain the password. See Molly, Adam, or Karin for access.
+  1. `$cp keystore.properties.example platforms/android/keystore.properties`
+  1. Replace the two password lines in `platforms/android/keystore.properties` with the actual
+  password. No space between the `=` sign and the password! This file is also gitignored. Please
+  do not put the real password in the `.example` file and push it to github.
+  1. Now you can build a release APK that should be signed automatically:
+  `$ionic cordova build android --prod --release`
+The terminal output should tell you your release location. If it's something like this:
+`/platforms/android/build/outputs/apk/release/android-release.apk`
+you should be fine.
+"android-release" and not "android-release-unsigned."
+that's a pretty good indication that you do not have a signed release APK.
+  1. In the [Google Play Developer Console](play.google.com/apps/publish), follow the instructions for uploading a new APK.
```

Comment:
Maybe add the steps for verifying the signature?

Reply:
How did we do that again?

Reply:
Um... maybe check your `history`? I don't remember exactly.

Reply:
So it was the command line? That's enough for me to go on.